packages/databyss-ui/components/Editor/state/reducer.js

```diff
@@ -154,6 +156,35 @@ const backspace = (state, payload) => {
   return cleanUpState(_state)
 }
 
+const getMarkupValues = (nextState, blockValue) => {
```

Comment:
We shouldn't be doing the conversion of `stateToSlate` or `slateToState` anywhere in the `state/reducer`. By this point the marks should be converted to a `ranges` key on the payload